packages/firestore/src/api/database.ts

```diff
@@ -307,13 +308,16 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
   // are already set to synchronize on the async queue.
   private _firestoreClient: FirestoreClient | undefined;
 
+  private isClientRunning: boolean;
```

Comment:
Boolean variables tend to named without the leading "is" (see e.g. fromCache, ssl, etc).